test/decl/var/property_wrappers.swift

```diff
@@ -1013,5 +1018,16 @@ struct MissingPropertyWrapperUnwrap {
     c(self.usesProjectedValue) // expected-error {{cannot convert value 'usesProjectedValue' of type 'W' to expected type 'V', use wrapper instead}}{{12-12=$}}
     d(self.$usesProjectedValue) // expected-error {{cannot convert value '$usesProjectedValue' of type 'V' to expected type 'W', use wrapped value instead}}{{12-13=}}
     d(self._usesProjectedValue) // expected-error {{cannot convert value '_usesProjectedValue' of type 'Baz<W>' to expected type 'W', use wrapped value instead}}{{12-13=}}
+
+    self.x["ultimate question"] // expected-error {{referencing subscript 'subscript(_:)' requires wrapper 'Foo<Int>'}} {{10-10=_}}
+    self.x["ultimate question"] = 42 // expected-error {{referencing subscript 'subscript(_:)' requires wrapper 'Foo<Int>'}} {{10-10=_}}
+  }
+}
+
+struct InvalidPropertyDelegateUse {
+  @Foo var x: Int = 42 // expected-error {{extra argument 'initialValue' in call}}
+
+  func test() {
+    self.x.foo() // expected-error {{alue of type 'Int' has no member 'foo'}}
```

Comment:
I believe this is a typo: `alue` -> `value`. Not sure how this was passing, is this test actually getting run?

Reply:
Ah thanks for noticing! {{}} represent regex so it would still much :)

Reply:
Ahh, gotcha, that makes sense. Never realized they represent regex as I've never needed to do any matching beyond exact text.